R/server.R

```diff
@@ -590,6 +600,156 @@ shinyServer(function(input, output, session) {
         removeModal()
     }, ignoreInit = TRUE)
     
+    
+    #### -------------------------- SLIP DAYS ----------------------------####
+
+    slip_days <- reactiveValues(
+        table = list(),
+        edit = NULL
```

Comment:
What's `edit = NULL`?

Reply:
this checks if you are editing a slip day policy and if so, lets you change the name of that policy and also throws an error if you try to rename the policy to a policy that exists.